src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -29,7 +29,7 @@ import { selectIsSnapshotApiQueriesPending, selectVotingPower } from 'state/apis
 import {
   selectHasUserEnteredAmount,
   selectInputSellAsset,
-  selectIsAccountMetadataLoading,
+  selectIsAccountsMetadataLoading,
```

Comment:
Renamed across the board. In a follow-up, we should probably introduce the notion of accounts granular state on top of/instead of the notion of all loaded.